app/values/iiif_path.rb

```diff
@@ -9,6 +9,10 @@ def to_s
     "#{iiif_url}/#{escaped_path}"
   end
 
+  def thumbnail
+    "#{self}/full/100,/0/default.jpg"
```

Comment:
To replicate the behavior we have now, we would want `!200,150` -- but I would also be fine with `200,` to be more compatible with other image servers.